api/v1/helpers/nouns/perspectives.js

```diff
@@ -64,6 +64,7 @@ module.exports = {
     owner: 'owner',
     lens: 'lens',
   },
+  fieldsToExclude: ['createdBy', 'ownerId', 'lensId'],
```

Comment:
Not really related to this PR but I'm curious.. Are we depreciating the createdBy field and using owner instead, or is there a difference between these?

Reply:
We are not deprecating `createdBy`.
`createdBy` is the user who created the record, but we are introducing a new concept of "owner" because the user who creates a record can come and go, change roles, etc., and we need the ability to reassign "ownership" of the record.